yellowbrick/utils/nan_warnings.py

```diff
@@ -0,0 +1,31 @@
+"""
+Small helpers that help find missing data.
+"""
+
+import numpy as np
+import warnings
+
+
+def warn_if_nans_exist(data):
+    """Warn if nans exist in a numpy array."""
+    null_count = count_nan_rows(data)
+    total = len(data)
+    percent = null_count / total
+
+    # if np.isnan(data).any():
+    if null_count > 0:
+        warnings.warn(
+            'Warning! Found {} rows of {} ({}%) with nan/null/None '
+            'values which are cannot be plotted.'.format(null_count, total,
+                                                         percent))
```

Comment:
Percent is not multiplied by 100, though the format string uses `{}%`, so the result will look something like `0.3%` instead of `30%` if I'm not mistaken ... ? 

Also, with floats we should probably limit the no of decimal places, e.g. `"{:0.2f}%"`